### PR TITLE
(GH-36) Installation Scripts use WScript Note

### LINKED ISSUE #36: Note: "WScript."
We need to followup to see what the user is attempting to do.

### PR DESCRIPTION
**Pull request by gep13**
Closes #36

### CODE REVIEW

File: src/chocolatey.package.validator/infrastructure.app/rules/InstallScriptsUseWscriptNote.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+namespace chocolatey.package.validator.infrastructure.app.rules
+{
+    using System.IO;
+    using infrastructure.rules;
+    using NuGet;
+
+    public class InstallScriptsUseWscriptNote : BasePackageRule
+    {
+        public override string ValidationFailureMessage { get { return "Package installation scripts use Wscript.  Is this really necessary?  Please see [here](https://github.com/chocolatey/package-validator/wiki/UsageOfWScript) for further information and guidance."; } }
```
Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/InstallScriptsUseWscriptNote.cs]: Notes are usually for the moderator / reviewer. Phrase the message appropriately. Also the here link.

**Comment by ferventcoder** [comment]
:+1:

**Comment by gep13** [comment]
@ferventcoder updated with suggested changes.

**Comment by ferventcoder** [comment]
Requires rebase.

**Comment by gep13** [comment]
@ferventcoder rebased.

**Pull request merged**